docs/faq.md

```diff
@@ -580,11 +623,11 @@ You can do that via the modifiers.ini file. Below is not a comprehensive list of
 No, Multitroid is a mod made by MilesTheNerd. While we greatly enjoy the mod and Miles is part of our team, it is not officially supported at present due to the time and energy required to provide official multiplayer support.
 
 #### How do I host on Multitroid?  
-1. You need to have the AM2RServer downloaded: <https://github.com/lassiterm/AM2R-Server/releases/latest>
-2. You need to either port forward, or use a VPN (like Hamachi, ZeroTier, Radmin etc.).
+1. Download AM2RServer at <https://github.com/lassiterm/AM2R-Server/releases/latest>
```

Comment:
```suggestion
1. Download the corresponding AM2RServer version at <https://github.com/lassiterm/AM2R-Server/releases/latest>
```
Also, something I got reminded of: there's an unoffocial multitroid update version thing, with its own server. that maybe should get mentioned as well?

Reply:
Well, there are multiple. There is one version for horde, one for fair-team, one for unoficial updates and I think one or two more as well?
Was just food for thought, considering there are so many of them it may be better for them to get skipped.

Reply:
Let's skip for now, I'll just commit your suggestion.